calibrators/marker_radar_lidar_calibrator/README.md

```diff
@@ -0,0 +1,141 @@
+| `use_radar_initial_crop_box_filter`         | `bool`        | `True`                                                  | Enables or disables the initial cropping filter for radar data processing.                                                                        |
+| `radar_initial_crop_box_min_x`              | `double`      | `-50.0`                                                 | Minimum x-coordinate for the initial radar calibration area.                                                                                      |
+| `radar_initial_crop_box_min_y`              | `double`      | `-50.0`                                                 | Minimum y-coordinate for the initial radar calibration area.                                                                                      |
+| `radar_initial_crop_box_min_z`              | `double`      | `-50.0`                                                 | Minimum z-coordinate for the initial radar calibration area.                                                                                      |
+| `radar_initial_crop_box_max_x`              | `double`      | `50.0`                                                  | Maximum x-coordinate for the initial radar calibration area.                                                                                      |
+| `radar_initial_crop_box_max_y`              | `double`      | `50.0`                                                  | Maximum y-coordinate for the initial radar calibration area.                                                                                      |
+| `radar_initial_crop_box_max_z`              | `double`      | `50.0`                                                  | Maximum z-coordinate for the initial radar calibration area.                                                                                      |
+| `lidar_background_model_leaf_size`          | `double`      | `0.1`                                                   | Voxel size in meter for the lidar background model.                                                                                               |
+| `radar_background_model_leaf_size`          | `double`      | `0.1`                                                   | Voxel size in meter for the radar background model.                                                                                               |
+| `max_calibration_range`                     | `double`      | `50.0`                                                  | Maximum range for calibration in meters.                                                                                                          |
+| `background_model_timeout`                  | `double`      | `5.0`                                                   | The background model will terminate if it is not updated within this period, measured in seconds.                                                 |
```

Comment:
I don't think the meaning of updated conveyed here well enough. If I remember correctly, it was update=new points in the background model

Reply:
fixed in https://github.com/tier4/CalibrationTools/pull/164/commits/1f6fc3365f891d5f2298cf7f1ae4a54db8ffa60a